src/static/css/cf-enhancements.less

```diff
@@ -246,18 +246,21 @@
   tags:
     - cfgov-cf-enhancements
 */
-
+@tr-border: @gray-50;
 th,
 td {
     padding: unit(10px / @base-font-size-px, em)
              unit(10px / @base-font-size-px, em);
-    background: @white;
     vertical-align: top;
 
     // Needed to override styles from cf-core.
+    // TODO: See about removing nth-child selector
```

Comment:
This area is really weird, because:

```
     tbody > tr:nth-child(odd) > & {
         background: inherit;
     }
    tbody > tr:nth-child(even) > & {
        background: inherit;
    }
```

…and…

```
     tbody > tr > & {
         background: inherit;
     }
```

Don't behave the same.